miasm2/core/parse_asm.py

```diff
@@ -51,61 +86,62 @@ def replace_expr_labels(expr, symbol_pool, replace_id):
     replace_id[expr] = m2_expr.ExprId(new_lbl, expr.size)
     return replace_id[expr]
 
+
 def replace_orphan_labels(instr, symbol_pool):
     """Link orphan labels used by @instr to the @symbol_pool"""
 
     for i, arg in enumerate(instr.args):
         replace_id = {}
-        arg.visit(lambda e:replace_expr_labels(e,
-                                               symbol_pool,
-                                               replace_id))
+        arg.visit(lambda e: replace_expr_labels(e,
+                                                symbol_pool,
+                                                replace_id))
         instr.args[i] = instr.args[i].replace_expr(replace_id)
 
 
+STATE_NO_BLOC = 0
+STATE_IN_BLOC = 1
+
+
+def parse_txt(mnemo, attrib, txt, symbol_pool=None):
+    """Parse an assembly listing. Returns a list of blocks and its associated
+    symbol_pool
```

Comment:
To be a little more precise, you could say "Returns a couple `(blocks, symbol_pool)`, where `blocks` is a list of `asm_bloc` and `symbol_pool` the associated `asm_symbol_pool`